### PR TITLE
doc: disambiguate sols vs. lamports in doco

### PR DESCRIPTION
**Pull request by sunnygleason**

### CODE REVIEW

File: src/connection.js [hunk 1]
```diff
@@ -535,7 +535,7 @@ export class Connection {
   }
 
   /**
-   * Fetch the current total currency supply of the cluster
+   * Fetch the current total currency supply, in lamports, of the cluster
```
Comment (CriesofCarrots) [inline src/connection.js]: Reads a little easier:
Fetch the current total currency supply of the cluster, in lamports

**Comment by t-nelson** [comment]
Whoops!  That was my negligence... Monkey see, monkey do. :)

Thanks for fixing!

**Comment by mvines** [comment]
semantic-release police caught ya:
```
doc: trivial (hopefully) disambiguate sols vs. lamports in doco...`
```

File: src/connection.js (hunk 1 above)
Comment (sunnygleason) [reply src/connection.js]: Fixed. I promise not to tell where I copied that from... 😂 

https://github.com/sunnygleason/solana-web3.js/blame/patch-1/src/connection.js#L43

Comment (mvines) [reply src/connection.js]: Gimme a break, I'm from 🇨🇦

Comment (CriesofCarrots) [reply src/connection.js]: That one reads okay... different sentence construction :)

**Pull request merged**